gr-uhd/apps/uhd_app.py

```diff
@@ -110,7 +112,7 @@ def get_usrp_info_string(self,
             if compact:
                 tpl = COMPACT_TPL
             return tpl.format(**info_pp)
-        except:
+        except RuntimeError:
```

Comment:
So the reason that I went for the runtime error was because that captures most of the information that we can get back from the Swig call, primarily the NotImplemented situation.  If we have a type issue at this interface we likely want to fix that.

Reply:
Went with Exception for now.  There is no logging support or real exception handling code in this app, it could use some re-factoring, but that is for another time.